setup.cfg

```diff
@@ -6,9 +6,6 @@ long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/pfmoore/editables
 author = Paul Moore
-author-email = p.f.moore@gmail.com
-maintainer = Paul Moore
-maintainer-email = p.f.moore@gmail.com
```

Comment:
Did the auto-formatter do this? I'm not keen, I'd rather keep this data together so that if any edits are needed it's easy to see that they are needed in multiple places. I'll take a look at the auto-formatter and see what config options it has.

Reply:
Yes; @asottile might know what's the reason for the order not being kept.

Reply:
ah, it moved these because it doesn't know about the dashed spelling -- the canonical spelling for setuptools is `author_email` and `maintainer_email`